scripts/src/primazactl/kube/roles/primazaroles.py

```diff
@@ -52,6 +52,10 @@ def get_primaza_namespace_role(role_name: str,
                 resources=["configmaps"],
                 verbs=["get", "list", "update", "delete"],
                 resource_names=["primaza-agentapp-config"]),
+            client.V1PolicyRule(
```

Comment:
This will break test-users. Need add the permissions to create the new policy role to the tenant: 
`scripts/src/primazatest/users/config/tenant.yaml`